src/test/java/resources/featurefile/shoppingCartPage.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Shopping cart link test
+  As a user I should navigate to shopping cart and verify the page
+
+  @smoke
+  Scenario: User should be able to navigate to shopping cart link
+    Given I am on HomePage page
+    When I click on shopping cart link
```

Comment:
Your code is not on same line .Please keep same distance .

Reply:
resolved